docs/source/developers/python.rst

```diff
@@ -306,6 +303,11 @@ adding flags with ``ON``:
 Anything set to ``ON`` above can also be turned off. Note that some compression
 libraries are needed for Parquet support.
 
+To enable C++ debugging information, pass the option ``-DCMAKE_BUILD_TYPE=debug``.
+
+.. seealso::
+   :ref:`cpp-building-building`.
```

Comment:
Can you add a label to that section? (I think we are now using an auto-generated one based on the title of the section)

So with the `.. _label:` syntax, like you also added a label for the extra-debugging section.